disnake/invite.py

```diff
@@ -543,11 +547,15 @@ def __str__(self) -> str:
         return self.url
 
     def __repr__(self) -> str:
-        return (
-            f"<Invite code={self.code!r} guild={self.guild!r} "
-            f"online={self.approximate_presence_count} "
-            f"members={self.approximate_member_count}>"
-        )
+        s = f"<Invite code={self.code!r} type={self.type!r}"
+        if self.type is not InviteType.friend:
```

Comment:
Should this be `if self.type is InviteType.guild`?

Reply:
That's better, yup. https://github.com/DisnakeDev/disnake/pull/1142/commits/93b205010502c1b25836b3e6eac4b6fd3dd1c6c4
I kept the member count in the repr for group dm invites, since the API apparently returns that here as well